### PR TITLE
More custom loadout fixes and additions

### LINKED ISSUE #2622: [NEWMAP] Two loadout items not working.
There are a two loadout items that cannot be selected even if you have enough points.

- Natural Philosopher's Wig
- Peoples Republic Medical Coat

### PR DESCRIPTION
**Pull request by Alberyk**
Fixes #2622
-adds more clothing for people to play dress up with their spessmen.
-fixes some long description on the ian's shirt
-allows people to toggle night vision glasses and material scanners while wearing them
-tweaks more marking related things

### CODE REVIEW

**Comment by Lohikar** [comment]
Does not compile.

**Comment by Lohikar** [comment]
Tested it, seems to work fine. Would like it you could move mannequins + the mannequins list into SSmob though, like so:

(in `code/controllers/subsystem/mob.dm`)
```
/datum/controller/subsystem/mobs/proc/get_mannequin(ckey)
	. = mannequins[ckey]
	if (!.)
		. = new /mob/living/carbon/human/dummy/mannequin
		mannequins[ckey] = .
```

File: code/modules/client/preference_setup/loadout/loadout_xeno.dm [hunk 1]
```diff
@@ -130,8 +130,8 @@
 	sort_category = "Xenowear"
 	
 /datum/gear/suit/tajaran_labcoat
-	display_name = "people's republic medical coat (Tajara)"
+	display_name = "peoples republic medical coat (Tajara)"
```
Comment (skull132) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm]: > Skull132 - Today at 11:28 AM
> Is it the People's Republic or Peoples Republic for Tajaran lore?

> Jenpai Sackboot - Today at 11:28 AM
> Why?
> People's is grammatically correct

Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm]: It is related to the bug. You can't select them in the loadout if the display name has an apostrophe.

Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_xeno.dm]: Replaced it with PRA.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 2]
```diff
@@ -50,17 +50,17 @@
 	path = /obj/item/clothing/head/beret/purple
 
 /datum/gear/head/beret/sec
-	display_name = "beret,security"
```
Comment (Lohikar) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: Won't changing these remove them from people's loadouts?

Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: Probably, but, not really an issue when a lot of loadout and preferences changes are happening with this update.

**Pull request merged**